README.md

```diff
@@ -114,9 +114,9 @@ are names of package managers. The values are dictionaries where the keys
 are default environment variables to set for that package manager and the
 values are the environment variable values.
 * `gomod_download_max_tries` - a maximum number of attempts for retrying go commands.
-* `gomod_strict_vendor` - the bool to disable/enable the strict vendor mode. For a repo that has gomod dependencies,
-if the `vendor` directory exists and this config option is set to `True`, one of the
-[vendoring flags](gomod.md#vendoring) must be used.
+* `gomod_strict_vendor` - (deprecated) was used to disable/enable the strict vendor mode. Now, the strict mode is
+enforced, which means that if the `vendor` directory exists, its contents must be up-to-date with what `go mod vendor`
```

Comment:
nitpick: strict mode defaulted to True, so there really isn't much a change, but the current wording kinda implies there is, if you just add (Default was True) somewhere in the first sentence, then it's all fine.